var/spack/repos/builtin/packages/laghos/package.py

```diff
@@ -36,6 +37,11 @@ class Laghos(MakefilePackage):
     depends_on('mfem@3.3.1-laghos-v1.0:+mpi+metis', when='@1.0,1.1+metis')
     depends_on('mfem@3.3.1-laghos-v1.0:+mpi~metis', when='@1.0,1.1~metis')
 
+    # 3.0 requirements
+    depends_on('hypre@2.11.2', when='@3.0:')
+    depends_on('mfem@develop+mpi+metis', when='@3.0:+metis')
+    depends_on('mfem@develop+mpi~metis', when='@3.0:~metis')
```

Comment:
I think you can remove the following lines above:
```python
depends_on('mfem@develop+mpi+metis', when='@develop+metis')
depends_on('mfem@develop+mpi~metis', when='@develop~metis')
```
`master` and `develop` both sort as higher than `@3.0:`

Reply:
The idea was to eventually figure out their actual requirements regarding mfem and specify that for v3 but with the amount of overhaul required on this there is no reason to duplicate logic now.

Fixed